extension.json

```diff
@@ -104,7 +104,8 @@
 				"DBLoadBalancer",
 				"WatchedItemStore",
 				"Fandom\\Includes\\User\\UserInfo",
-				"Fandom\\Includes\\Util\\UrlUtilityService"
+				"Fandom\\Includes\\Util\\UrlUtilityService",
+				"Reverb\\Notification\\NotificationBroadcastFactory"
```

Comment:
Note: This issue was unexpected for me - when HookHandler class cannot be created, it fails silently (at least in this case)
```
ArgumentCountError from line 56 of /extensions/Reverb/src/ReverbBroadcastHookHandler.php: Too few arguments to function Reverb\ReverbBroadcastHookHandler::__construct(), 11 passed in /vendor/wikimedia/object-factory/src/ObjectFactory.php on line 247 and exactly 12 expected
	#0 /vendor/wikimedia/object-factory/src/ObjectFactory.php(247): Reverb\ReverbBroadcastHookHandler->__construct(GlobalVarConfig, MediaWiki\User\UserFactory, MediaWiki\Revision\RevisionStore, NamespaceInfo, MediaWiki\Revision\RevisionStore, MediaWiki\User\UserOptionsManager, MediaWiki\Languages\LanguageFactory, Fandom\Includes\Database\FandomLoadBalancer, Fandom\Includes\Watchlist\WatchedItemStoreOverrides, Fandom\Includes\User\UserInfo, Fandom\Includes\Util\UrlUtilityService)
	#1 /vendor/wikimedia/object-factory/src/ObjectFactory.php(152): Wikimedia\ObjectFactory\ObjectFactory::getObjectFromSpec(array, array)
	#2 /includes/HookContainer/HookContainer.php(473): Wikimedia\ObjectFactory\ObjectFactory->createObject(array)